src/selection_indicator/mod.rs

```diff
@@ -200,15 +199,16 @@ where
             Size::new(display_size.width, selected_item_height),
         );
 
-        let invert_area = self.style.draw(
+        let _invert_area = self.style.draw(
             &menu_state.indicator_state.state,
             input_state,
-            &mut display.cropped(&selected_item_area),
+            &mut display.cropped(&selected_item_area).color_converted(),
         )?;
 
-        // Translate inverting area to its position
-        let invert_area = invert_area.translate(selected_item_area.top_left);
-        let mut inverting = display.invert_area(&invert_area);
+        // // TODO: Figure out how to handle inversion
```

Comment:
I think we could move this problem into the Theme. Some themes may not need any color manipulation and it may make readability worse. They could define a noop conversion method.

Inverting is just making the text visible under a background that is filling in with the same color on a BinaryColor display. I think we could:

 - keep text rendering BinaryColor and treat the text it draws as a mask
 - modify the adapter to take two colors
 - map these two colors to the text's On and Off colors, or flip the mapping over the inverting area

Reply:
> Inverting is just making the text visible under a background that is filling in with the same color on a BinaryColor display.

I wonder if it is really more simple than this - and simply swapping the color of the background with the color of the text. The mask approach feels unnecessarily complex, unless there is a desire to support menus with transparent backgrounds. Personally, that feels a bit more like a niche use case but I also admit I don't currently have any grasp on how this project may be currently used by others in this regard.

Reply:
I'm not sure what you're arguing for here. This "inversion" that is now commented out is BinaryColor-specific: if you have a text color, and a background color, then if the background color changes under the text, the text color has to be inverted for the text to be readable.

With a color display, inversion is:
 - not always the best option
 - not always necessary as you can have two different background colors and a third one for the text that looks always good

An adapter that would map the text pixels vs empty pixels to one color on the other would support both use cases. Text is already drawn as a bitmask mapped to different colors by embedded-graphics, just without any "background-awareness" that we need here, meaning that sticking to BinaryColor as our text renderer's direct output would save an intermediate conversion.

Reply:
What I was suggesting is that if a menu's color is stylized through something akin to a "theme" (as mentioned elsewhere in this PR), and the "theme" specifies both a text color and a background color, an ignorant process of "inversion" becomes just swapping these two when drawing the specified `MenuItem`. 

I suspect if the "theme" approach is taken that it may just make more sense to specify a "selected color", or such.